pyproject.toml

```diff
@@ -2,12 +2,21 @@
 line-length = 120
 include = '\.pyi?$'
 
-[tool.isort]
-profile = 'black'
-line_length = 120
-multi_line_output = 3
-include_trailing_comma = true
-float_to_top = true
+[tool.ruff]
+# Adds to default excludes: https://ruff.rs/docs/settings/#exclude
+extend-exclude = [
+    "django-source",
+    "stubgen",
+    "out",
+]
+line-length = 120
+target-version = "py38"
+select = [
+    "I",    # isort
+]
+
+[tool.ruff.isort]
+split-on-trailing-comma = false
```

Comment:
This option makes Ruff collapse multiline imports back to a single line if they're short enough to fit. E.g.

```python
from typing import (
    Any,
)
```
to
```python
from typing import Any
```

Reply:
Related: https://github.com/charliermarsh/ruff/issues/4153